examples/DefaultBoard/DefaultBoard.ino

```diff
@@ -70,10 +70,13 @@ void loop() {
     board.processChar(newChar);
   }
 
-  // Used to abort multi part messages
+ /* this is no longer required with new muli char messages
+  *  
+ // Used to abort multi part messages
```

Comment:
this is supposed to be:

```Arduino
if (board.isMultiCharCmd) {
  board.checkMultiCharCmdTimer();
}
```

and then `checkMultiCharCmdTimer()` should send a timeout message. Please do not cause breaking changes please see the function you are whining out and verify your functions return those to the user. https://github.com/aj-ptw/OpenBCI_32bit_Library/blob/dev-3.0.0/OpenBCI_32bit_Library.cpp#L151

Reply:
Got it